scripts/known_good.json

```diff
@@ -23,10 +23,6 @@
       ],
       "cmake_options" : [
         "-DBUILD_TESTS=OFF"
-      ],
-      "build_platforms" : [
```

Comment:
Good catch! I totally missed this because I share dependencies between different projects and they were already built